src/CartBlock.C

```diff
@@ -700,8 +702,16 @@ void CartBlock::writeCellFile(int bid)
   ibmax=-30000000;
   nnodes=(dims[1]+1)*(dims[0]+1)*(dims[2]+1);
   ncells=dims[0]*dims[1]*dims[2];
-  sprintf(intstring,"%d",100000+myid);
-  sprintf(fname,"cart_cell%s.dat",&(intstring[1]));
+
+  std::ostringstream ss;
```

Comment:
Is `snprintf` an option?

Reply:
I replaced sprintf with snprintf in PR #34 (https://github.com/Exawind/tioga/pull/34), but I was still getting errors.

Reply:
> hmmm we go from 2 lines of code to 8. Kinda too bad. Is there a way to consolidate all this? Make a helper function? Inline some things instead of creating new variables? Or maybe we need to keep sprintf. Or skip the stream thing and just cast straight to string?

I could write a function that accepts a name root, a number, and a file type, and then returns the full file name. That's probably the way to go to make this tidier.

Reply:
yeah it seems like a lot of the same things. A function would be nice.